Sources/QRCodeReader.swift

```diff
@@ -78,44 +78,71 @@ public final class QRCodeReader: NSObject, AVCaptureMetadataOutputObjectsDelegat
   /// Block is executed when a metadata object is found.
   public var didFindCode: ((QRCodeReaderResult) -> Void)?
 
-  // MARK: - Creating the Code Reader
-
+  // MARK: - Creating the Code Reade
+  
   /**
    Initializes the code reader with the QRCode metadata type object.
    */
   public convenience override init() {
-    self.init(metadataObjectTypes: [AVMetadataObjectTypeQRCode])
+    self.init(startingCaptureDevicePosition: .back)
+  }
+  
+  /**
+   Initializes the code reader with the starting capture device position, and the default array of metadata object types
+   
+   - parameter startingCaptureDevicePosition: The capture position to use on start of scanning
+   */
+  public convenience init(startingCaptureDevicePosition position: AVCaptureDevicePosition) {
+    self.init(startingCaptureDevicePosition: position, metadataObjectTypes: [AVMetadataObjectTypeQRCode])
+  }
+  
+  /**
+   Initializes the code reader with an array of metadata object types, and the default initial capture position
+   
+   - parameter metadataObjectTypes: An array of strings identifying the types of metadata objects to process.
+   */
+  public convenience init(metadataObjectTypes types: [String]) {
+    self.init(startingCaptureDevicePosition: .back, metadataObjectTypes: types)
   }
 
   /**
    Initializes the code reader with an array of metadata object types.
-
+   
+   - parameter startingCaptureDevicePosition: The Camera to use on start of scanning
    - parameter metadataObjectTypes: An array of strings identifying the types of metadata objects to process.
    */
-  public init(metadataObjectTypes types: [String]) {
+  public init(startingCaptureDevicePosition: AVCaptureDevicePosition, metadataObjectTypes types: [String]) {
```

Comment:
Add the default parameter value and invert the argument position to avoid breaks:

```swift
public init(metadataObjectTypes types: [String] = [AVMetadataObjectTypeQRCode], captureDevicePosition: AVCaptureDevicePosition = .back) {
```